### PR TITLE
List classes renaming

### PR DESCRIPTION
**Pull request by kurpav**

### CODE REVIEW

File: src/styles/blocks/list.less [hunk 1]
```diff
@@ -1,205 +1,209 @@
...
-    .dd__list__heading__section__item {
-      display: table-cell;
-      vertical-align: middle;
-      padding: 0 15px;
+        //overwrites
+        .dd__checkbox.dd__checkbox--checked .checkbox__label {
+          color: @dusty-gray;
+        }
+
+        .checkbox__label {
```
Comment (okolomiets) [inline src/styles/blocks/list.less]: https://jira.doubledutch.me/browse/STEW-273
could you check it out and fix?

Comment (kurpav) [reply src/styles/blocks/list.less]: fixed

File: src/styles/blocks/list.less [hunk 2]
```diff
@@ -1,205 +1,211 @@
...
-      //overwrites
-      .dd__checkbox.dd__checkbox--checked .checkbox__label {
-        color: @dusty-gray;
-      }
+      &.dd__list-section--active {
+        color: @shakespeare;
 
-      .checkbox__label {
-        margin-left: 36px;
+        .dd__list-section-item {
```
Comment (freeman14) [inline src/styles/blocks/list.less]: dd__list-section__item

Comment (kurpav) [reply src/styles/blocks/list.less]: fixed

**Pull request merged**